frontend/packages/knative-plugin/src/yaml-templates.ts

```diff
@@ -0,0 +1,20 @@
+import { Map as ImmutableMap } from 'immutable';
+import { ServiceModel } from './models';
+
+export const yamlTemplates = ImmutableMap().setIn(
+  [ServiceModel, 'default'],
+  `
+apiVersion: ${ServiceModel.apiGroup}/${ServiceModel.apiVersion}
+kind: ${ServiceModel.kind}
+metadata:
+  name: example
+  namespace: default
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: docker.io/imagename
```

Comment:
can we use an existing sample image name ex. openshift/hello-openshift?

Reply:
Done